sift/packages/ntfs-3g.sls

```diff
@@ -0,0 +1,2 @@
+ntfs-3g:
```

Comment:
Adjust per first comment on review. Needs to be `sift-package-ntfs-3g` with modifications to `pkg.installed` instruction to reference actual name.